src/tcgwars/logic/impl/gen8/DarknessAblaze.groovy

```diff
@@ -2955,7 +2960,94 @@ public enum DarknessAblaze implements LogicCardInfo {
         weakness F
         bwAbility "Infinity Zone", {
           text "If all of your Pokémon in play are [D], you may now have up to 8 [D] Pokémon on your Bench and can’t play any other type of Pokémon. (When this Ability no longer works, discard your Benched Pokémon until there are 5 left.)"
-          actionA {
+          bwAbility "Infinity Zone", {
```

Comment:
`bwAbility "Infinity Zone"` already is there.

Reply:
Thanks, nice catch